client/src/app/gateways/repositories/meeting-repository.service.ts

```diff
@@ -72,7 +72,9 @@ export class MeetingRepositoryService extends BaseRepository<ViewMeeting, Meetin
             `admin_group_id`,
             `committee_id`,
             `group_ids`,
-            `language`
+            `language`,
+            `user_ids`,
```

Comment:
Why do you need `user_ids` here? If you need it is there any way to prevent loading this? 

Keep in mind that these are fields for a list containing all meetings and `user_ids` is a potentially very big field.

Reply:
Ah, yes. I used the group way to get the meeting membership, so these two user_ids are no longer needed.